azure-communication-ui/build.gradle

```diff
@@ -32,12 +32,40 @@ buildscript {
         if (project.rootProject.file('private_drop.properties').canRead()) {
             Properties privateDropProps = new Properties()
             privateDropProps.load(project.rootProject.file('private_drop.properties').newDataInputStream())
-            azure_calling_sdk_version = privateDropProps.getProperty('PRIVATE_DROP_VERSION', '""')
+            azure_calling_sdk_version_ga = privateDropProps.getProperty('PRIVATE_DROP_VERSION', '""')
+            azure_calling_sdk_version_beta = azure_calling_sdk_version_ga
         }
         else {
-            azure_calling_sdk_version = '2.6.0-beta.6'
+            // To use Beta Calling SDK and Features, set the following in local.properties:
+            //
+            // USE_CALLING_SDK_BETA=true
+            //
+            // and then run ./gradlew clean
+            // This will use the beta version of the SDK
+            // and enable the beta features
+
+            // In this branch, it was already Beta version, so it'll be in the GA field here
+            // This should be set to a GA release before merge back
+            azure_calling_sdk_version_ga = '2.6.0-beta.6'
```

Comment:
If we read from local.properties anyway to find out which SDK version to use, why not just keep a single azure_calling_sdk_version variable and read from local.properties an override the azure_calling_sdk_version defined in build.gradle.
So if we add azure_calling_sdk_version to the local.propertyies, then use that value.

Reply:
meant to be pinned to latest public release, aligning with Calling SDK
i.e. they always have a latest GA and latest Beta. This should generally be these values here.

We can add a version over-ride to local.properties if people can use. but not in this PR.